source/postcard/src/ser/flavors.rs

```diff
@@ -250,6 +250,47 @@ where
     }
 }
 
+/// Wrapper over a [`core::iter::Extend<u8>`] that implements the flavor trait
+///
+/// This is like [`ExtendFlavor`], but it takes the writer by reference instead of
+/// by value. This allows you to remain in control of the writer.
+pub struct ExtendRefFlavor<'a, T> {
```

Comment:
Maybe use the word "mut" instead? When I see "ref", I usually expect `&`.
```suggestion
pub struct ExtendMutFlavor<'a, T> {
```